### ISSUE TITLE
Release artifacts inconsistent with semantic versioning

### ISSUE THREAD

**Issue by nethershaw** [efmer/boinctasks-js#77]
Releases seem to follow semantic versioning, i.e., major.minor; artifacts seem to introduce an arbitrary separator (e.g., 1.23 -> 1.2.3) in the filename that is inconsistent with the release version. This doesn't make sense and complicates efforts to package the software for other distributions. Please reconcile them.

**Comment by efmer** [comment]
I use revisions.
The first digit only changes on a breaking huge design change.
The number after the . is sequential.
You can strip out the last . as I do.
Program versions are downward compatible most of the time, but not upward.
So you can go from 1.00 -> 1.24 but not from 1.23 -> 1.00 without the chance of breaking one on the storage files.
I do not use major minor artifacts, I gave up deciding what is major or minor.

**Comment by nethershaw** [comment]
...why bother displaying x.y.z in one place and x.yz in another, then, if you mean no difference at all between the two? You seem to have misunderstood me; I am not talking about moving between versions, upgrading, or downgrading. I am talking about how you label a _single version_ two different ways: 1.33 here, and 1.3.3 inside the package.

1.3.3 is _not_ the same thing as 1.33 in packaging systems because the dot is always a delimiter with a very specific meaning. Your comment hasn't made it any clearer which of the two forms you really mean to use and which should be displayed in packaging systems (Debian will say 1.3.3 while all websites will say 1.33, which _will_ cause some people to think they are not up to date).

Other packaging systems (e.g., Portage) can't even find your artifacts unless we program in a transformation from your release version to your artifact version because they aren't the same and everyone's first assumption is that they should be.

TL;DR: The dot matters. It doesn't matter whether you implement major/minor versioning. Just decide on _one_ way to version your application and use it exactly the same way everywhere, please.

**Comment by nethershaw** [comment]
You are using NPM. NPM forces semver on you. If you don't use semver, NPM tells you the version in package.json is invalid. Sticking an arbitrary dot between two digits just so you have enough dots is the wrong way to satisfy what it's telling you to do.

Semver is `MAJOR.MINOR.PATCH`.

Suggestion: if you don't care about distinguishing between MINOR and PATCH (this is what you have been doing), just always set PATCH to zero and be done with it.

**Thus, in package.json, `version` should be `1.33.0`; not `1.3.3`.**
